src/pages/topics/[slug].tsx

```diff
@@ -25,6 +36,45 @@ const Topic: NextPage<TopicProps> = ({ name, description, ctas }) => {
                 description={description}
                 ctas={ctas}
             />
+            <div
+                style={{
+                    display: 'flex',
+                    gap: '20px',
+                    flexWrap: 'wrap',
+                }}
+            >
+                {contentCategories.map(c =>
```

Comment:
nitpick but can we be more explicit when naming the arguments here so rather than just `c` we could have a `category`?